pypesto/objective/amici_util.py

```diff
@@ -286,7 +286,7 @@ def get_error_output(
         amici_model: AmiciModel,
         edatas: Sequence['amici.ExpData'],
         rdatas: Sequence['amici.ReturnData'],
-        sensi_order: int,
+        sensi_orders: Tuple[int],
```

Comment:
I think `Tuple[int, ...]` would be more correct https://docs.python.org/3/library/typing.html#typing.Tuple